src/components/atoms/UserAvatar/UserAvatar.tsx

```diff
@@ -23,7 +23,7 @@ export const UserAvatar: React.FC<UserAvatarProps> = ({
   containerClassName,
   imageClassName,
   onClick,
-  isOnline,
```

Comment:
`isOnline` is defined by us. We check if the user has been online lately. We should keep it

Reply:
I explained this above. Please, don't silently unresolve the conversations